samples/tmart/tmart-portal/main.bal

```diff
@@ -0,0 +1,52 @@
+import ballerina/log;
+import ballerina/http;
+import ballerinax/jaeger as _;
+
+type POrder record {
+    string orderId;
+    string customerId;
+    string productId;
+    int quantity;
+};
+
+configurable string shipmentHost = "localhost";
+configurable string inventoryHost = "localhost";
+configurable string crmHost = "localhost";
+configurable string bookParkHost = "localhost";
+
+public function main() {
+    log:printInfo("Starting the TMART portal...");
+}
+
+service /tmart on new http:Listener(9100){
+    resource function post orders(POrder porder) returns json|error {
+        log:printInfo("Order received: " + porder.orderId);
+        do {
+            http:Client customers = check new("http://" + crmHost + ":9102/customers");
```

Comment:
Any specific reason to create client wihtin resource which is creating a client with each new request?